plugins/GSdx/GSdx.rc

```diff
@@ -333,15 +334,14 @@ BEGIN
         LEFTMARGIN, 6
         RIGHTMARGIN, 242
         TOPMARGIN, 7
-        BOTTOMMARGIN, 244
```

Comment:
Why removed?

Reply:
Unsure, I never touched that dialog. I fixed it.